### ISSUE TITLE
[Feature request] paste snippet on file creation

### ISSUE THREAD

**Issue by BachiMjavanadze** [mre/vscode-snippet#608]
I found 2 amazing extensions: [Auto Snippet](https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.auto-snippet) and [Smart File Templates](https://marketplace.visualstudio.com/items?itemName=TrevorNesbitt.smart-file-templates).  they have the feature that definitely should have your extention. These extensions give ability to automatically paste snippets on file creation:

![img](https://github.com/mre/vscode-snippet/assets/38082501/40443d16-080c-48c7-8d40-861d796eebcc)

I asked them to support synchronization feature but problem is that either they are abandoned or have bad support.  Maybe you will copy this feature and implement my [issue](https://github.com/tnesbitt210/smart-file-templates/issues/7)?

**Comment by VladBrok** [comment]
Yes this is cool. I won't be able to implement it any time soon because I have many tasks on other projects. 

I've added the "good first issue" label. External contributors are welcome!

I'll leave an implementation note: this should be configurable, because not all snippets and not for all languages are suitable for this.

**Comment by BachiMjavanadze** [comment]
I will update my request;
Just do not look at `Smart File Templates` extension because [Auto Snippet](https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.auto-snippet) is almost done; It has a sync feature as it is configured in `settings.json` and uses regular `VSCode` snippets (which sync by default). But it does not support multiple snippets for the same file; For this case, a dropdown menu is needed... a dropdown menu is needed in any case, as I explained above

**Comment by BachiMjavanadze** [comment]
no need this feature anymore, this is better:  https://marketplace.visualstudio.com/items?itemName=BachiMjavanadze.terminal-gui
